scripts/airdrop.js

```diff
@@ -21,13 +24,55 @@ const argv = default_yargs
     describe: "Do not actually send transactions, just simulate their submission",
   }).argv
 
+const parseCsvFile = async function (filePath) {
+  return new Promise((resolve, reject) => {
+    const results = []
+    fs.createReadStream(filePath, { encoding: "utf-8" })
+      .pipe(csv())
+      .on("data", (data) => results.push(data))
+      .on("end", () => resolve(results))
+      .on("error", (error) => reject(error))
+  })
+}
+
+const toPayment = function (leaderBoardItem) {
+  const { receiver, amount, token_address: tokenAddress } = leaderBoardItem
+
+  return {
+    tokenAddress,
+    receiver,
+    amount: new BigNumber(amount),
+  }
+}
+
+const parseTransferFile = async function (filename) {
+  const ext = path.extname(filename).toLowerCase()
+
+  if (ext === ".csv") {
+    const results = await parseCsvFile(filename)
+
+    const payments = results.map(toPayment).filter((payment) => !payment.amount.isZero())
```

Comment:
Should we do this filtering outside of the parsing, so that it will be applied to both formats?

Reply:
Yes! I will fix this.